### PR TITLE
Feat/crud students classroom

### PR DESCRIPTION
**Pull request by pikaden**
Login with account: 
user@example.com
string

Go to http://localhost:3000/classrooms/6/manage
User can remove 1 user
image

User can remove batch user
image

User can search

TODO:
classroom header
invite student by code, by email
navigation menu

### CODE REVIEW

**Comment by pikaden** [comment]
Need to authorization: there are 2 options:
1. Only teacher can access, and he can invite other student
2. Member and teacher can access, but only teacher can invite, default is isStudentAllowInvite

Header is only show classroom detail, and no button invite student in this table

File: src/app/[locale]/(main)/classrooms/[id]/manage/actions/get-all-members-action.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  const url = getAPIServerURL(`/classrooms/get-all-member/${classroomId}?` + params)
+
+  const res: PagedResponse<ClassroomMembers> = await fetch(url, option)
+    .then((res) => res.json())
+    .catch((err) => {
+      console.error(`[ERROR] getClassroomMembers: ${url} `, err)
+      throw new Error(err)
+    })
+  return res
```
Comment (blackmouse572) [inline src/app/[locale]/(main)/classrooms/[id]/manage/actions/get-all-members-action.ts:42]: Following fetch backend pattern to make sure same result for every page. Ref for get-all-quizbank

File: src/types/classroom.ts [hunk 2]
```diff
@@ -23,8 +23,22 @@ type Classroom = {
...
+  email: string
+  avatar: string
+  dob: string
+  useAICount: number
+  favoriteBankIds: number[]
+  created: string
+  updated: string
+  isVerified: boolean
+}
+
```
Comment (blackmouse572) [inline src/types/classroom.ts:39]: Isn't same as User type ?

**Review by blackmouse572 — CHANGES_REQUESTED** [review]
Nice work, just little adjustments

**Review by blackmouse572 — APPROVED** [review]

**Pull request merged**